odoo/addons/base/models/ir_module.py

```diff
@@ -111,7 +111,14 @@ class MyFilterMessages(Transform):
 
     def apply(self):
         for node in self.document.traverse(nodes.system_message):
-            _logger.warning("docutils' system message present: %s", str(node))
+            # Some docutils messages are only informative; this is the case for
+            # "Duplicate explicit target name" - see ``docutils/nodes.py`` >
+            # ``set_duplicate_name_id``: level=1 can be used for an info message.
+            level = node.get("level")
+            logger_method = "info" if level is not None and level <= 1 else "warning"
+            getattr(_logger, logger_method)(
+                "docutils' system message present: %s", str(node)
```

Comment:
Thanks for the review, yeah I saw https://runbot.odoo.com/runbot/build/50893767 but figured it would be allowed here as this is not user input

I'll test & update the PR

Reply:
Works fine, and on second thought it makes more sense - use method pointers instead of strings...

Pushed